test/conformance/kuttl/generate/validation/clusterpolicy/permissions/same-kind/policy-1-subresource.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: generate-same-kind-pol-1
+spec:
+  rules:
+    - name: generate-add-labels-policy
+      match:
+        all:
+        - resources:
+            kinds:
+              - Namespace
+      generate:
+        synchronize: true
+        apiVersion: kyverno.io/v1
+        kind: Policy/status
```

Comment:
Why using a subresource? The referenced issue was about just generating the parent `Policy` resource.

Reply:
My mistake, didn't read through carefully enough.